book/src/configuration.md

```diff
@@ -1 +1,81 @@
 # Configuration
+
+## Theme
+
+Use a custom theme by placing a theme.toml in your config directory (i.e ~/.config/helix/theme.toml). The default theme.toml can be found [here](https://github.com/helix-editor/helix/blob/master/theme.toml), and user submitted themes [here](https://github.com/helix-editor/helix/blob/master/contrib/themes).
+
+Styles in theme.toml are specified of in the form:
+
+```toml
+"key" = { fg = "#ffffff", bg = "#000000", modifiers = ["bold", "italic"] }
+```
+
+where `name` represents what you want to style, `fg` specifies the foreground color, `bg` the background color, and `modifiers` is a list of style modifiers. `bg` and `modifiers` can be omitted to defer to the defaults.
+
+To specify only the foreground color:
+
+ ```toml
+ "key" = "#ffffff"
+ ```
+
+ Possible modifiers:
+
+| modifier |
```

Comment:
Looking at the rich diff, these tables don't render for me correctly: https://github.com/helix-editor/helix/blob/f5883bbe7f3bf854c274706a75b66f24fb7a6736/book/src/configuration.md